resources/zib/zib-TNMTumorClassification.TPrimaryTumor.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-TNMTumorClassification.TPrimaryTumor" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-TNMTumorClassification.T_PrimaryTumor" />
```

Comment:
In the profiling guidelines it is stated that the `.id` coincides with the latter part of the `.url`, but that is not the case here.